youtube_dl/extractor/godtv.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+from .common import InfoExtractor
+from .ooyala import OoyalaIE
+
+
+class GodTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?god\.tv(?:/[a-z0-9-]+)+/(?P<display_id>[a-z0-9-]+)'
```

Comment:
Use more relaxed version, e.g. `[^/]+` to match segments and `[^/?#&]+` to match id.